.gitmodules

```diff
@@ -0,0 +1,18 @@
+[submodule "mtb-pdl-cat1"]
+	path = mtb-pdl-cat1
+	url = https://github.com/Infineon/mtb-pdl-cat1.git
+[submodule "mtb-hal-cat1"]
+	path = mtb-hal-cat1
+	url = https://github.com/Infineon/mtb-hal-cat1.git
+[submodule "core-lib"]
+	path = core-lib
+	url = https://github.com/Infineon/core-lib.git
+[submodule "psoc6cm0p"]
+	path = psoc6cm0p
+	url = https://github.com/Infineon/psoc6cm0p.git
+[submodule "abstraction-rtos"]
+	path = abstraction-rtos
+	url = https://github.com/Infineon/abstraction-rtos.git
+[submodule "whd-bsp-integration"]
+	path = whd-bsp-integration
+	url = https://github.com/Infineon/whd-bsp-integration.git
```

Comment:
why are you adding submodules here ?

Are you planning to fetch those through the west manifest ?
Cause if that is the case, then please read this section:
https://docs.zephyrproject.org/latest/guides/modules.html#module-repositories
> Module Repositories
>
>  - All modules included in the default manifest shall be hosted in repositories under the zephyrproject-rtos GitHub organization.

Would say the direction proposed here should be discussed at dev-review / TSC.

Reply:
updated, now we do not use submodule for hal_infineon.